vetiver/templates/model_card.qmd

```diff
@@ -134,7 +137,7 @@ dmp
 ## Visualize model performance
 
 ```{python}
-performance = px.scatter(sacramento, x="price", y = "preds", facet_row="type", trendline="ols")
```

Comment:
removing trendline to avoid forcing a `statsmodels` install, which is not a required package in vetiver